WordPress/src/main/java/org/wordpress/android/ui/reader/utils/ReaderUtils.java

```diff
@@ -247,6 +247,18 @@ public static long getBlogIdFromBlogPreviewUrl(String url) {
         }
     }
 
+    public static boolean isTagUrl(String url) {
```

Comment:
💡 Would it make sense to add a unit test using this method so that if anyone changes the RegEx the tests will fail? We already have a `ReaderUtilsTest` class.

Reply:
Good point, I somehow missed this unit test 🤦